packages/desktop-client/src/components/reports/reports/CustomReport.tsx

```diff
@@ -387,6 +387,7 @@ export function CustomReport() {
   ]);
 
   const getGraphData = useMemo(() => {
+    // TODO: fix me - state mutations should not happen inside `useMemo`
```

Comment:
Is this something you want to fix in this PR or is it being flagged to be completed in a different PR?

Reply:
That's for a different PR. I _might_ pick it up when integrating dashboards more tightly with custom reports. But I might also not. Either way - still good to call it out so eventually it's patched.